integreat_compass/cms/views/offers/offer_form_view.py

```diff
@@ -0,0 +1,158 @@
+import logging
+
+from django.contrib import messages
+from django.db import transaction
+from django.shortcuts import render
+from django.utils.decorators import method_decorator
+from django.utils.translation import gettext as _
+from django.views.generic import TemplateView
+
+from ...decorators import permission_required
+from ...forms import (
+    ContactForm,
+    LocationForm,
+    OfferForm,
+    OfferVersionForm,
+    OrganizationForm,
+)
+from ...models import Offer
+
+logger = logging.getLogger(__name__)
+
+
+@method_decorator(permission_required("cms.edit_offer"), name="dispatch")
```

Comment:
```suggestion
@method_decorator(permission_required("cms.change_offer"), name="dispatch")
```
I think we could also remove the custom permission `edit_offer`, since it essentially means the same thing as the default permission `change_offer`, right?